oak_functions/loader/src/server.rs

```diff
@@ -461,19 +447,44 @@ impl wasmi::Externals for WasmState {
                     .read_extension_args(request_ptr, request_len)
                     .map_err(|err| {
                         self.log_error(&format!(
-                            "Handle {:?}: Unable to read input from guest memory: {:?}",
-                            extension.get_handle(),
+                            "Unable to read input from guest memory: {:?}",
                             err
                         ));
                         OakStatus::ErrInvalidArgs
                     });
 
+                let extension = &mut self
+                    .extensions_indices
+                    .get_mut(&index)
+                    .unwrap_or_else(|| panic!("Unimplemented function at {}", index));
```

Comment:
Could you confirm it's fine to panic here, i.e. it's not possible for untrusted Wasm code to cause this panic? I would still always worry about this every time I read this code, so perhaps worth leaving a comment to explain this, or, better, avoid the panic altogether.

Reply:
This code will disappear, once we call every extension through invoke. Are you against keeping it as it is? I don't see immediately how this could be triggered from the Wasm code.

Reply:
SG